### DISCUSSION TITLE
Allow Sharing / Subscribing / Cloning / Watching / etc.

### DISCUSSION THREAD

**Discussion by MarkMoretto** [community/community#34661] [Other Feature Feedback, Questions, & Ideas]
Hi,

I'm not sure if the idea of having list subscribers / followers, sharing lists, cloning lists, etc., is something that has been posted before, but it might be interesting to test out.

AFAIK, the URL for a list comes up as something like:

`https://github.com/stars/{github-username}/lists/{name-of-list}`

But, when I navigate to the URL when not logged in, there's no other immediate option to follow a list.

I wouldn't say that this kind of feature is super top priority, but something more for the back burner.

Cheers,

- mm

**Comment by Adzkiya768** [comment]
Hello master can you help me master

**Reply by Adzkiya768** [reply]
hello, may I ask for help, brother, I need a script because I have a project that hasn't been completed yet.

**Comment by jfullstackdev** [comment]
if you are not log in, like in Incognito, you don't have the identity, without that, you can't follow, comment, etc.,

**Reply by buttsons-mart** [reply]
![1dd2e02624bde65a069371bb896ca0a5.jpg](https://user-images.githubusercontent.com/115039543/194224284-484d29a7-7c2c-43bb-ae4c-83f53ab56d6b.jpg)
